edge/msghub/cpu2msghub/Dockerfile.amd64

```diff
@@ -1,5 +1,5 @@
 FROM alpine:latest
-RUN apk --no-cache --update add curl ca-certificates wget jq bc
+RUN apk --no-cache --update add curl ca-certificates wget jq bc bash
```

Comment:
Why did you add bash? I'm not seeing other changes that require it. An edge service best practice is to keep the images as small as possible.

Reply:
Oh shoot, I forgot to remove that. I added it to test with the new gps so I could exec into the cpu2msghub container and curl gps manually. I'll get rid of it!